src/Sarif.Sarifer.Core/BackgroundAnalyzerBase.cs

```diff
@@ -201,7 +201,7 @@ private static async Task<string> GetSolutionDirectoryAsync()
                 dataToInsert: OptionallyEmittedData.ComprehensiveRegionProperties | OptionallyEmittedData.TextFiles | OptionallyEmittedData.VersionControlDetails,
                 dataToRemove: OptionallyEmittedData.None,
                 tool: this.MakeTool(),
-                levels: new List<FailureLevel> { FailureLevel.Error, FailureLevel.Warning, FailureLevel.Note, FailureLevel.None },
+                levels: new List<FailureLevel> { FailureLevel.Error }, // Options: FailureLevel.Warning, FailureLevel.Note, FailureLevel.None.
```

Comment:
>// Options: FailureLevel.Warning, FailureLevel.Note, FailureLevel.None. [](http://example.com/codeflow?start=71&length=71)

why remove these levels?
if we need support Pass result, when kind = Pass, all levels will be Note. it will filter out these pass results